src/contractTest/java/uk/gov/hmcts/dm/controller/DocumentStoreProviderTest.java

```diff
@@ -65,6 +67,14 @@ void pactVerificationTestTemplate(PactVerificationContext context) {
         }
     }
 
+    @PactBrokerConsumerVersionSelectors
```

Comment:
Can we not just extend src/contractTest/java/uk/gov/hmcts/dm/controller/provider/BaseProviderTest.java? to inherit this method?